docs/covidcast-py/plot_directive/plot_examples-1.py

```diff
@@ -0,0 +1,6 @@
+import covidcast
```

Comment:
are these some new way that sphinx handles plot examples? I haven't seen this before

Reply:
I think each code chunk in the docs comes with a link to the plot output and the source code as a file, so you can easily reuse it